hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DiskBalancerManager.java

```diff
@@ -340,9 +306,13 @@ public void processDiskBalancerReport(DiskBalancerReportProto reportProto,
     long failureMoveCount = reportProto.getFailureMoveCount();
     long bytesToMove = reportProto.getBytesToMove();
     long balancedBytes = reportProto.getBalancedBytes();
+    double volumeDataDensity = reportProto.hasVolumeDataDensity() ? 
```

Comment:
Since disk balancer is in feature branch, we don't need to handle the proto backward compatibility so far. So no need to call reportProto.hasVolumeDataDensity() to detect whether report has this field.